src/ol/renderer/canvas/VectorTileLayer.js

```diff
@@ -365,7 +365,7 @@ CanvasVectorTileLayerRenderer.prototype.handleStyleImageChange_ = function(event
 CanvasVectorTileLayerRenderer.prototype.postCompose = function(context, frameState, layerState) {
   const layer = this.getLayer();
   const declutterReplays = layer.getDeclutter() ? {} : null;
-  const source = /** @type {ol.source.VectorTile} */ (layer.getSource());
+  const source = /** @type {module:ol/source/VectorTile~VectorTileSource} */ (layer.getSource());
```

Comment:
This is not consistent with the rest of the library. It's probably better to change the `VectorTile` import name in `src/ol/source/VectorTile` to `Tile`, and name the constructor `VectorTile`.

Reply:
> and name the constructor `VectorTile`.

which one? in `ol/renderer/canvas/VectorTileLayer` (this file) ?

Reply:
No, sorry for being unspecific. In `src/ol/source/VectorTile.js`.